src/CMakeLists.txt

```diff
@@ -1,21 +1 @@
-add_library(maxminddb
-  maxminddb.c
-  data-pool.c
-)
-
-if(WIN32)
-  target_link_libraries(maxminddb ws2_32)
-endif()
-
-set_target_properties(maxminddb
-  PROPERTIES PUBLIC_HEADER
-  "../include/maxminddb_config.h;../include/maxminddb.h"
-  SOVERSION ${MAXMINDDB_SOVERSION}
-)
-
-install(
-  TARGETS maxminddb
-  RUNTIME DESTINATION bin
-  LIBRARY DESTINATION lib
-  ARCHIVE DESTINATION lib
-)
+# Dummy file
```

Comment:
I think we can get rid of this file and remove `src/CMakeLists.txt` from the `EXTRA_DIST` section of `Makefile.am`. I am pretty sure that is why the CodeQL build was failing.